src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -4,17 +4,34 @@
 
 package org.carlmontrobotics.subsystems;
 
+import org.carlmontrobotics.Constants;
 import org.carlmontrobotics.Constants.Arm.*;
+import org.carlmontrobotics.lib199.MotorConfig;
+import org.carlmontrobotics.lib199.MotorControllerFactory;
 
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
 import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.motorcontrol.MotorController;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+
 
 public class Arm extends SubsystemBase {
+    private final CANSparkMax armMotor1 = MotorControllerFactory.createSparkMax(Constants.Arm.MOTOR_PORT1,MotorConfig.NEO);
+    //private final CANSparkMax armMotor2 = MotorControllerFactory.createSparkMax(Constants.Arm.MOTOR_PORT2,MotorConfig.NEO);
+    //there is only one arm motor. 
+    private final SimpleMotorFeedforward armFeed = new SimpleMotorFeedforward(Constants.Arm.kS, Constants.Arm.kV);
+    private final PIDController armPID = new PIDController(kP[ARM], kI[ARM], kD[ARM]);
```

Comment:
Already asked this in Constants.java but why are PID values stored in an array? Also, you never imported PIDController and the variables can't be resolved. MAKE SURE CODE BUILDS BEFORE PUSHING!!